### PR TITLE
Introduction of poddisruptionbudget for openmetadata server

### PR DESCRIPTION
**Pull request by preetsshah**
### Describe your changes :
I worked on the Pod Disruption Budget for Openmetadata server to ensure availability of the server pod.

#
### Type of change :

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Reviewers
@akash-jain-10

### CODE REVIEW

File: charts/openmetadata/values.yaml [hunk 1]
```diff
@@ -389,5 +389,11 @@ startupProbe:
     path: /healthcheck
     port: http-admin
 
+podDisruptionBudget:
+  enabled: true
```
Comment (akash-jain-10) [inline charts/openmetadata/values.yaml:393]: By default, it should be enabled.
```suggestion
  enabled: false
```

File: charts/openmetadata/values.yaml [hunk 2]
```diff
@@ -389,5 +389,11 @@ startupProbe:
...
+  config:
+    maxUnavailable: 1
+    # minAvailable: 1
```
Comment (akash-jain-10) [inline charts/openmetadata/values.yaml:396]: This should not be commented out as it will be a required property when enabled right ?

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]
Thanks for the Contribution. Have left a couple of reviews as well as this PR is missing the updated on JSON Schema of Helm Values addition.

Please refer to this link for more context - https://helm.sh/docs/topics/charts/#schema-files

File: charts/openmetadata/values.yaml (hunk 2 above)
Comment (preetsshah) [reply charts/openmetadata/values.yaml:396]: Correct, un-commenting this one in the following commit.

File: charts/openmetadata/values.yaml (hunk 1 above)
Comment (preetsshah) [reply charts/openmetadata/values.yaml:393]: Changing the code to have PDB disabled by default.

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**